app/javascript/app/routes.js

```diff
@@ -162,10 +164,53 @@ export default [
       {
         path: '/about',
         component: About,
-        exact: true,
         nav: true,
         label: 'ABOUT',
-        headerImage: 'about'
+        headerImage: 'about',
+        routes: [
+          {
+            path: '/about',
+            component: AboutDescription,
+            exact: true,
+            anchor: true,
+            label: 'About Climate Watch'
+          },
+          {
+            path: '/about/partners',
+            component: AboutPartners,
+            exact: true,
+            anchor: true,
+            label: 'Climate Watch Partners'
+          },
+          {
+            path: '/about/contact',
```

Comment:
I think we might have to hide the tabs that have no content yet. Let's check this on Monday morning.

Reply:
@Bluesmile82 let me know when you're online and we can go over the other tabs and see which ones to add.

Reply:
Looking at the file let's add the "Permissions & Licensing" Tab, as we seem to have the right content for that. I would also add the Contact Us and Feedback one.

We can use the same form that we are using here: http://vizzuality.github.io/climate-watch-splash , (click the sign up button). So it's an iframe showing up in a modal window.

Reply:
I agree with Simao hiding the empty sections... should be enough setting the anchor to false in the routes config.
Related to the Permissions and licensing we should propose to move this into the footer
![image](https://user-images.githubusercontent.com/10500650/32170135-b90855e2-bd73-11e7-90d3-a21d459b7680.png)